integrations/SnapPixel/util.js

```diff
@@ -158,6 +158,41 @@ const ecommEventPayload = (event, message) => {
       };
       break;
     }
+    case "product viewed":
+      payload = {
+        ...payload,
+        item_ids: get(message, "properties.product_id"),
```

Comment:
Shouldn't we store the product_id in an array and then set it to item_ids?
CC @utsabc

Reply:
Did this as it was not an array of ids, it was just one but we can store it in an array as Snapchat has represented it as an array.

Reply:
Added